pkg/reconciler/instances/eventing/preaction/removenatsoperatorstep_test.go

```diff
@@ -2,64 +2,80 @@ package preaction
er\n  namespace: kyma-system\n---\n# Source: nats/templates/20-service.yaml\napiVersion: v1\nkind: Service\nmetadata:\n  name: eventing-nats\n  labels: \n    helm.sh/chart: nats-1.0.0\n    app: nats\n    app.kubernetes.io/managed-by: Helm\n    kyma-project.io/dashboard: eventing\nspec:\n  ports:\n  - name: tcp-client\n    port: 4222\n    targetPort: client\n  selector: \n    app: nats\n---\n# Source: nats/templates/10-deployment.yaml\napiVersion: apps/v1\nkind: Deployment\nmetadata:\n  name: nats-operator\n  # Change to the name of the namespace where to install NATS Operator.\n  # Alternatively, change to \"nats-io\" to perform a cluster-scoped deployment in supported versions.\nspec:\n  replicas: 1\n  selector:\n    matchLabels:\n      name: nats-operator\n  template:\n    metadata:\n      annotations:\n        sidecar.istio.io/inject: \"false\"\n      labels:\n        name: nats-operator\n    spec:\n      serviceAccountName: nats-operator\n      containers:\n      - name: nats-operator\n        image: \"eu.gcr.io/kyma-project/nats-operator:c33012c2\"\n        imagePullPolicy: \"IfNotPresent\"\n        args:\n        - nats-operator\n        # Uncomment to perform a cluster-scoped deployment in supported versions.\n        #- --feature-gates=ClusterScoped=true\n        ports:\n        - name: readyz\n          containerPort: 8080\n        env:\n        - name: MY_POD_NAMESPACE\n          valueFrom:\n            fieldRef:\n              fieldPath: metadata.namespace\n        - name: MY_POD_NAME\n          valueFrom:\n            fieldRef:\n              fieldPath: metadata.name\n        readinessProbe:\n          httpGet:\n            path: /readyz\n            port: readyz\n          initialDelaySeconds: 15\n          timeoutSeconds: 3\n---\n# Source: nats/templates/30-destination-rule.yaml\napiVersion: networking.istio.io/v1alpha3\nkind: DestinationRule\nmetadata:\n  name: eventing-nats\nspec:\n  host: eventing-nats.kyma-system.svc.cluster.local\n  trafficPolicy:\n    tls:\n      mode: DISABLE\n---\n# Source: nats/templates/40-cr.yaml\napiVersion: nats.io/v1alpha2\nkind: NatsCluster\nmetadata:\n  name: eventing-nats\nspec:\n  size: 1\n  version: \"2.1.8\"\n  serverImage: \"eu.gcr.io/kyma-project/external/nats\"\n  pod:\n    annotations:\n      sidecar.istio.io/inject: \"false\"\n    labels:\n      helm.sh/chart: nats-1.0.0\n      app: nats\n      app.kubernetes.io/managed-by: Helm\n      kyma-project.io/dashboard: eventing\n    resources:\n      limits:\n        cpu: 20m\n        memory: 64Mi\n      requests:\n        cpu: 5m\n        memory: 16Mi\n  natsConfig:\n    debug: true\n    trace: true\n  template:\n    spec:\n      affinity:\n        podAntiAffinity:\n          preferredDuringSchedulingIgnoredDuringExecution:\n            - podAffinityTerm:\n                labelSelector:\n                  matchLabels:\n                    nats_cluster: eventing-nats\n                topologyKey: kubernetes.io/hostname\n              weight: 100\n"
```

Comment:
Multiline strings would probably make this way more understandable/editable.

Reply:
Done